spec/models/piece_spec.rb

```diff
@@ -50,4 +50,73 @@
       king.errors.clear
     end
   end
+  describe "#move_to!" do
+    context "when the tile is empty" do
+      it "moves to the coordinates" do
+        board = create(:game)
+        board.pieces.delete_all
+        king = game.kings.first
+        king.update_attributes(x_position: 1, y_position: 1)
+        king.move_to!(2, 2)
+        expect(king.x_position).to eq(2)
+        expect(king.y_position).to eq(2)
+      end
+    end
+    context "when the tile is not empty" do
+      it "captures the opponent piece and moves to the new coordinates" do
+        board = create(:game)
+        board.pieces.delete_all
+        white_king = game.kings.first
```

Comment:
change this to white_king = King.create(x_position: 1, y_position: 1, game_id: board.id, color: true)